js/renovation/component_wrapper/editors/editor.ts

```diff
@@ -127,4 +127,4 @@ export default class Editor extends Component {
 
 const prevIsEditor = (OldEditor as unknown as { isEditor }).isEditor;
```

Comment:
Can not we specify the type here more strict? e.g `(OldEditor as unknown as  { isEditor: (instance: Component) => boolean })` or smth like this?

It will neutralize the warning you're trying to fix